internal/validators/kubescape.go

```diff
@@ -51,7 +54,8 @@ func (n *KubescapeService) Manifests() ([]kubescapev1.VulnerabilityManifest, err
 	return manifests, nil
 }
 
-func (n *KubescapeService) ReconcileSeverityRule(nn ktypes.NamespacedName, rule validationv1.SeverityLimitRule, ignoredCVEs []string, manifests []kubescapev1.VulnerabilityManifest) (*types.ValidationRuleResult, error) {
```

Comment:
Param `ignoredCVEs` was unused. The refactor went all the way up to spec. I figured it's best to reintroduce it to the spec after we implement it instead of keeping it in the spec but unused til then.